pkg/controller/registry/resolver/resolver.go

```diff
@@ -175,6 +176,23 @@ func (r *SatResolver) SolveOperators(namespaces []string, csvs []*v1alpha1.Clust
 	return operators, nil
 }
 
+func (r *SatResolver) toBundleInstallable(entry *cache.Entry) (*BundleInstallable, error) {
```

Comment:
I would like to see a comment here to explain what this func does and perhaps a better name for it as it is too vague. This func essentially adds cluster/system constraints to the installable bundles which is fine.

Reply:
I've had a crack at it. See if it's better.

Reply:
I think the new name is better.